### DISCUSSION TITLE
hwaccel

### DISCUSSION THREAD

**Discussion by fleischereiahmet** [stashapp/stash#4725] [Q&A] [answered]
Hey Guys can anybody help me with hwaccel in stash via LXC in Proxmox?
Things i have done so far:

- [x] passthrough iGPU to LXC
- [x] installed vainfo and intel-gpu-tools
- [x] turned on FFmpeg hardware encoding
- [x] edit FFmpeg Transcode Input Args with "-hwaccel
- [x] edit FFmpeg Transcode Output Args with "auto"

when i open the intel_gpu_top it seems like the iGPU is not transcoding at all but CPU is taking the main part. 

What am i´m doing wrong?

**Comment by feederbox826** [comment, marked as answer]
`intel-gpu-tools` is not enough for hwaccel - you need `i965-va-driver-shaders` (<gen8) OR `intel-media-va-drivers-non-free` (>gen8) as well as `intel-opencl-icd`  (required) These are both in `nonfree` and `nonfree-drivers` repositories that you need to add
